src/DesignRuns/device_planner_dialog.cpp

```diff
@@ -13,6 +15,13 @@ DevicePlannerDialog::DevicePlannerDialog(QWidget *parent)
   setWindowFlags(Qt::Dialog | Qt::WindowCloseButtonHint);
   setWindowTitle(tr("Device Planner"));
 
+  // size and positioning dialog
+  QScreen *screen = QGuiApplication::primaryScreen();
```

Comment:
This code was removed on purpose since it is obsolete

Reply:
got it, i removed it